docs/sql/statements/alter-table.rst

```diff
@@ -187,6 +187,17 @@ Can be used to drop a column from a table.
 
 It's possible to drop multiple columns at once.
 
+.. NOTE::
+
+    It's not allowed to drop a column which is part of a
+    :ref:`primary key <primary_key_constraint>`, used in
+    :ref:`CLUSTERED BY column <gloss-clustered-by-column>`,
+    used in :ref:`PARTITIONED BY <gloss-partitioned-by-column>`, or is
+    referenced in a :ref:`generated column <ddl-generated-columns-expressions>`.
```

Comment:
I think we also need to restrict dropping columns, used in FULLTEXT INDEX as a source.

Reply:
thx, added!